hawkular-wildfly-agent/pom.xml

```diff
@@ -113,6 +113,7 @@
     <dependency>
       <groupId>org.jboss.logging</groupId>
       <artifactId>jboss-logging</artifactId>
+      <version>3.1.4.GA-redhat-2</version>
```

Comment:
That also works, It's because EAP7 logging lib also has debugf(String format, Object param1); signature.
Added the explanation on the XML file.